dclab/rtdc_dataset/feat_temp.py

```diff
@@ -1,9 +1,10 @@
 """
 .. versionadded:: 0.33.0
 """
-from ..definitions import feat_logic
+import numpy as np
 
-from .fmt_hierarchy import RTDC_Hierarchy
+from ..definitions import feat_logic
+from .fmt_hierarchy import RTDC_Hierarchy, map_indices_child2root
```

Comment:
please keep the blank line between the `.` and `..` imports.

Reply:
Alright!